qa/tasks/cephfs/mount.py

```diff
@@ -110,12 +132,13 @@ def _setup_brx_and_nat(self):
         if not brx:
             log.info("Setuping the 'ceph-brx' with {0}/{1}".format(ip, mask))
 
-            args = ["sudo", "bash", "-c", "ip link add name ceph-brx type bridge"]
+            args = ['sudo', 'ip', 'link', 'add', 'name', 'ceph-brx', 'type', 'bridge']
+            self.client_remote.run(args=args, timeout=(5*60))
+            args = ['sudo', 'ip', 'addr', 'flush', 'dev', 'ceph-brx']
             self.client_remote.run(args=args, timeout=(5*60))
```

Comment:
Wouldn't it be bettter to do this in the teardown method?

Reply:
Do you mean the 'flush' here ?

Reply:
Yes.

Reply:
No, it won't work. 
This will trying to resolve the case:
When the test node was used to do some test cases and then after that the bridge will be removed, but the network config file will be left and if we reboot the node and run the test case again, when we setup the `ceph-brx` bridge again it will config it automatically by reading from the stale network config file.

Then if we config the bridge again it will fail.